src/common/library.c

```diff
@@ -93,6 +90,13 @@ SASSERT(_Alignof(FARPROC) == _Alignof(void*));
 SASSERT(sizeof(HMODULE) == sizeof(void*));
 SASSERT(_Alignof(HMODULE) == _Alignof(void*));
 
+static void OS_libfmt(char* buffer, i30 size, const char* name)
+{
+    StrCpy(buffer, size, name);
```

Comment:
This can just be SPrintf(buffer, size, "%s.dll", name); for windows, and SPrintf(buffer, size, "lib%s.so"); for posix. No more need for LIB_PRE or LIB_EXT : )